gateway-ha/src/main/java/io/trino/gateway/ha/resource/EntityEditorResource.java

```diff
@@ -86,8 +91,13 @@ public Response updateEntity(
                     ProxyBackendConfiguration backend =
                             OBJECT_MAPPER.readValue(jsonPayload, ProxyBackendConfiguration.class);
                     gatewayBackendManager.updateBackend(backend);
-                    log.info("Setting up the backend %s with healthy state", backend.getName());
+                    log.info("Turning cluster %s %s", backend.getName(), backend.isActive() ? "on" : "off");
                     routingManager.updateBackEndHealth(backend.getName(), backend.isActive());
+                    backendStateManager.updateStates(
```

Comment:
this is needed to update the states on the UI. We can refactor to get rid of BackendStateManager in the future. https://github.com/trinodb/trino-gateway/issues/359

Reply:
how does the ui change after adding this state, pending? can you share screenshots?

Reply:
UI currently doesn't show the healthstate of each cluster. I can submit a PR to add that onto UI after this one